src/Game.cpp

```diff
@@ -31,6 +31,164 @@
 #include "UnkObjects/unk6C2E88.hpp"
 #include "UnkObjects/unk6C3250.hpp"
 #include "UnkObjects/unk6C3890.hpp"
+#include "UnkObjects/unk8600F8.hpp"
+#include "config.hpp"
+
+static void sub_612A80(const char *args) {
+    glo_8600F0 = 0;
+    glo_8600EC = nullptr;
+    char *esi = (char*)args;
+    if (*esi != 0) {
+        esi = (char*)std::strchr(args, ' ');
```

Comment:
Similar to my previous note, this is not a problem that will prevent me from merging, but I still want you to avoid in the future. Don't mix `char *` and `const char *`. The travis build log from #4 (which isn't there right now) was erroring/warning on these types of conversions, and I'd like to avoid them in the future.

In general, any string literals (which includes memory locations in the original game that contain a string already) are `const char *` (which is the same as `char const *`, but I generally write it the other way). Any variables, function arguments, and `glo_` variables that get assigned a `const char *` value at any point in the code should be declared as `const char *` as well.